conformance/results/mypy/directives_deprecated.toml

```diff
@@ -0,0 +1,20 @@
+conformant = "Unsupported"
+notes = """
+Does not support @deprecated
```

Comment:
Nit: Please include period at end of this sentence. This is displayed in the hover text within the conformance summary report.